benchmarking/scripts/processor.js

```diff
@@ -37,7 +37,7 @@ const nativeTokenTransfer = async (context, ee, next) => {
 
     // break to make sure transactions arrive in order to RPC.
     // NOTE: reduce it if you are skipping validate
-    await sleep(350);
+    await sleep(500);
```

Comment:
why 500? could this slow tps if 500>time to post tx

Reply:
So madara block time is 6second.

So, if means atleast with this we will send 12 transactions per block. So, if our TPS is less than let's say even 3, that means the pool has enough transactions, and they are not being processed!

Also, the script waits for some time for pool to be filled with transactions before collection any data.